SQLiteTests/AggregateFunctionsTests.swift

```diff
@@ -1,5 +1,5 @@
 import XCTest
-import SQLite
+@testable import SQLite
```

Comment:
I generally avoid `@testable` in frameworks till I need it. I try to test the public API only. Were you encountering an issue running tests without this attribute?